projects/client/RabbitMQ.Client/src/client/impl/AutorecoveringModel.cs

```diff
@@ -822,6 +827,12 @@ public void BasicCancel(string consumerTag)
                 body);
         }
 
+        public void UpdateSecret(string newSecret, string reason)
+        {
+            m_delegate.BasicPublish(newSecret,
```

Comment:
Should it call `m_delegate.UpdateSecret`?

Reply:
Why do channels (`IModel`s) need to be aware of secrets? They don't handle authentication. Can this be fully encapsulated in a connection?

Reply:
This method is not unused. Will remove.